openmdao/docs/openmdao_book/features/core_features/options/options.ipynb

```diff
@@ -503,12 +503,212 @@
    "source": [
     "assert(msg == \"Option 'size' with value 5 must be an even number.\")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting Multiple Option Values Simultaneously\n",
+    "\n",
+    "The *OptionsDictionary* class is versatile and can be used in a variety of situations.\n",
+    "While all systems contain an `options` attribute, users may wish to introduce other options for their systems or other classes using OpenMDAO's *OptionsDictionary*.\n",
+    "\n",
+    "After initialization, users might want the ability to modify more than one option at a time.\n",
+    "For this purpose, the `set` method on *OptionsDictionary* allows exists."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openmdao.api as om\n",
+    "\n",
+    "prob = om.Problem()\n",
+    "linear_comp = prob.model.add_subsystem('linear', LinearCombinationComp())\n",
+    "\n",
+    "# Set options a and b at the same time.\n",
+    "linear_comp.options.set(a=3., b=5.)\n",
+    "\n",
+    "prob.setup()\n",
+    "\n",
+    "prob.set_val('linear.x', 3)\n",
+    "\n",
+    "prob.run_model()\n",
+    "print(prob.get_val('linear.y'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "remove-input",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "assert(prob.get_val('linear.y') == 14.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting options throughout a problem model (`Problem.model_options`)\n",
+    "\n",
+    "It is common for options to be passed down through a model tree from parent Group to child Group or Component.\n",
+    "This can make the addition of a new option deep in the model a tedious procedure, since each parent system may need to have an option added to itself in order to know how to instantiate its children, and then actually take the given input option and pass it to the child.\n",
+    "\n",
+    "Different options are available to alleviate this. For defining the options, it might be useful to have a common function that is referenced by all pertinent classes which adds the necessary options. Alternatively, those classes can subclass from a class which automatically adds the appropriate options.\n",
+    "\n",
+    "For passing options to components nested deeply within the model, the OpenMDAO `Problem` object provides an attribute named `model_options`. `Problem.model_options` is a standard dictionary, keyed by a string that serves as a glob filter for system pathnames.\n",
```

Comment:
suggest hyperlinking "glob filter" to a definition, e.g. https://en.wikipedia.org/wiki/Glob_(programming)